.env

```diff
@@ -0,0 +1,4 @@
+MongoDB_user_name='HYF-Ammar'
```

Comment:
I would try to keep these environment variables with the same names as you are going to use in production.

For example, Heroku uses the env variable DATABASE_URL to save information about the database.

So you could have DATABASE_URL="mylocal database credentials". It would simplify a lot the deployment of the server